Cargo.toml

```
@@ -56,7 +56,7 @@ default = ["crypto-dependencies"]
 experimental = ["bitvec", "fiat-crypto", "fixed", "num-bigint", "num-rational", "num-traits", "num-integer", "num-iter", "rand"]
 multithreaded = ["rayon"]
 prio2 = ["crypto-dependencies", "hmac", "sha2"]
```

Since `crypto-dependencies` now pulls in `"hmac", "sha2"`, I think we can remove them from here?
```suggestion
prio2 = ["crypto-dependencies"]
```